checker/src/org/checkerframework/checker/index/samelen/SameLenAnnotatedTypeFactory.java

```diff
@@ -108,7 +114,30 @@ public AnnotationMirror createCombinedSameLen(
             bValues.addAll(SameLenUtils.getValue(sl2));
         }
 
-        return getCombinedSameLen(aValues, bValues);
+        AnnotationMirror res = getCombinedSameLen(aValues, bValues);
+        return res;
+    }
+
+    public FlowExpressions.Receiver arrayReceiverFromString(
```

Comment:
I had that thought while writing it. I wasn't sure of the answer and I figured you would comment on it, since I named it as if it needed an array and then didn't use the property. If we moved it into a superclass, that would allow MinLenAnnotatedTypeFactory#getMinLenFromString to call it as well (which is especially good, since the code in that method is all duplicated in getMinLenFromString, as well (or more accurately I stole the code from there...)).